src/model/naming_test.go

```diff
@@ -18,16 +20,17 @@ func TestDefaultNaming(t *testing.T) {
 				BranchKey: BranchKey{
 					Id: 1234,
 				},
-				Name: "my Super-BRANCH",
+				Name:      "my Super-BRANCH",
+				IsDefault: false,
```

Comment:
`IsDefault` sa minule presunulo do `Branch` struktury, tak nie je potrebne to posielat ako parameter, zmazane `false` nizsie.